integ/components/deadline/deadline_01_repository/lib/repository-testing-tier.ts

```diff
@@ -115,7 +115,7 @@ export class RepositoryTestingTier extends TestingTier {
       'cd ~ec2-user',
       'mkdir -p testScripts',
       'cd testScripts',
-      'wget https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem',
+      'wget https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem',
```

Comment:
We should probably call this out in the release notes / CR description too?

Reply:
I'll add this in

Reply:
Added mention of this to the commit notes. There's some mention of this in the CR description, since this is the code that uses the new CA cert rather than the expired 2019 one